lib/overcommit/hook_context/post_commit.rb

```diff
@@ -0,0 +1,20 @@
+module Overcommit::HookContext
+  # Contains helpers related to contextual information used by post-commit
+  # hooks.
+  class PostCommit < Base
+    # Get a list of files that were added, copied, or modified in the last
+    # commit. Renames and deletions are ignored, since there should be nothing
+    # to check.
+    def modified_files_last_commit
+      @modified_files_last_commit ||= Overcommit::GitRepo.modified_files_last_commit
+    end
+
+    # Returns the set of line numbers corresponding to the lines that were
+    # changed in a specified file.
+    def modified_lines_in_file_last_commit(file)
```

Comment:
Ditto on changing this to match the `PreCommit` hook context helper method, `modified_lines_in_file`.